digdag-standards/src/main/java/io/digdag/standards/command/ecs/EcsClientConfig.java

```diff
@@ -109,7 +120,7 @@ private static EcsClientConfig buildEcsClientConfig(String clusterName, Config e
                 .withCapacityProviderName(ecsConfig.getOptional("capacity_provider_name", String.class))
                 .withContainerCpu(ecsConfig.getOptional("container_cpu", Integer.class))
                 .withContainerMemory(ecsConfig.getOptional("container_memory", Integer.class))
-                .withStartedBy(ecsConfig.getOptional("startedBy", String.class))
+                .withStartedBy(ecsConfig.getOptional("started_by", String.class))
```

Comment:
Could you explain this change? May be for keeping naming rule.
Is there any compatibility issue (in our side)?

Reply:
Just typo

Reply:
Do you mean that this option is not used yet and no impact of this change ?

Reply:
Yes. This option is not yet used in our environment.